app/screens/AddChildSetup.tsx

```diff
@@ -0,0 +1,573 @@
+import { userRealmCommon } from '../database/dbquery/userRealmCommon';
+import { ChildEntity, ChildEntitySchema } from '../database/schema/ChildDataSchema';
+import { backup } from '../services/backup';
+import { addChild, apiJsonDataGet, getAge, getNewChild, isFutureDate, setActiveChild } from '../services/childCRUD';
+import { validateForm } from '../services/Utils';
+import {
+  Heading1Centerw,
+  Heading3,
+  Heading4Regularw,
+  ShiftFromTop20,
+  SideSpacing25,
+  Heading3Centerw,
+  Heading2w,
+  Heading1,
+  Heading4Regular,
+  ShiftFromTopBottom5,
+  Heading3w,
+  ShiftFromTop25,
+  Heading2Centerw,
+  Heading3BoldCenterrw,
+} from '../styles/typography';
+import AlertModal from '@components/AlertModal';
+import { BannerContainer } from '@components/shared/Container';
+import { SettingHeading, SettingShareData, SettingOptions } from '@components/shared/SettingsStyle';
+import VectorImage from 'react-native-vector-image';
+import useNetInfoHook from '../customHooks/useNetInfoHook';
+import DocumentPicker, { isInProgress } from 'react-native-document-picker';
+import * as ScopedStorage from "react-native-scoped-storage";
+import RNFS from 'react-native-fs';
+import TextInputML from '@components/shared/TextInputML';
+import { bgcolorWhite2, primaryColor } from '@styles/style';
+import AesCrypto from 'react-native-aes-crypto';
+import { encryptionsIVKey, encryptionsKey } from 'react-native-dotenv';
+import BackgroundColors from '@components/shared/BackgroundColors';
+import { logEvent } from '../services/EventSyncService';
+import { EXPECTED_CHILD_ENTERED } from '@assets/data/firebaseEvents';
+import { dataRealmCommon } from '../database/dbquery/dataRealmCommon';
+import { ConfigSettingsEntity, ConfigSettingsSchema } from '../database/schema/ConfigSettingsSchema';
+import { setAllLocalNotificationGenerateType } from '../redux/reducers/notificationSlice';
+import { setActiveChildData } from '../redux/reducers/childSlice';
+
+type ChildSetupNavigationProp = StackNavigationProp<
+  RootStackParamList,
+  'ChildSetupList'
+>;
+
+type Props = {
+  navigation: ChildSetupNavigationProp;
+};
+const styles = StyleSheet.create({
+  containerView: {
+    backgroundColor: bgcolorWhite2,
+    flex: 1
+  },
+  flex2Style: {
+    alignItems: 'flex-start'
+  },
+  flexRow1: {
+    marginTop: 10
+  },
+  scrollViewStyle: {
+    padding: 0,
+    paddingTop: 0
+  },
+  textInputStyle: {
+    width: '100%',
+  },
+  textParentInfoStyle: {
+    textAlign: 'center'
+  },
+  dividerStyle: {
+    marginEnd: 20
+  },
+  orDividerStyle: {
+    width: 172,
+    alignSelf: 'center',
+  },
+  uploadTextStyle: {
+    color: "#1CABE2"
+  },
+})
+const AddChildSetup = ({ route, navigation }: Props): any => {
+  const { t } = useTranslation();
+  const [relationship, setRelationship] = useState('');
+  const [userRelationToParent, setUserRelationToParent] = useState();
+  const [relationshipname, setRelationshipName] = useState('');

```

Comment:
Rename relationshipname to relationshipName

Reply:
We had already changed and code is merge with development branch.